test-data/genops-classes.test

```diff
@@ -62,3 +62,33 @@ L0:
     r5 = 1
     r6 = r4 + r5 :: int
     return r6
+
+[case testMethodCall]
+class A:
+    def f(self, x: int, y: str) -> int:
+        return x + 10
+
+def g(a: A) -> None:
+    a.f(1, 'hi')
+[out]
+def f(self, x, y):
```

Comment:
It would be a bit clearer if methods would have the class name prefixed, in particular as there can be multiple classes with the same method name. Maybe something like `def A.f(self, x, y):` or `class A:` followed by methods, with indentation.